app/admin/RTMediaFormHandler.php

```diff
@@ -355,6 +355,21 @@ static function display_render_options( $options ) {
 			),
 		);
 
+		if( function_exists( 'bp_is_active' ) ){
+			$render[ 'rtmedia-enable-masonry-activity' ] = array(
```

Comment:
remove space before/after `[]` like this `$render['rtmedia-enable-masonry-activity']`